### PR TITLE
Ms broadcast

### PR DESCRIPTION
**Pull request by dron0gus**

### CODE REVIEW

File: firmware/controllers/can/can_dash_ms.cpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+#if EFI_CAN_SUPPORT
+#include "can.h"
+#include "can_msg_tx.h"
+
+#include "rusefi_types.h"
+#include "rtc_helper.h"
+#include "fuel_math.h"
+
+/* TODO:
+ * use engine->outputChannels instead of Sensor::getOrZero as it cheaper */
```
Comment (rusefillc) [inline firmware/controllers/can/can_dash_ms.cpp:21]: no, no-no-no!!! Definitely not read from engine->outputChannels

Comment (dron0gus) [reply firmware/controllers/can/can_dash_ms.cpp:21]: Don't we care about coherency and cpu time? Why do same job twice?

Comment (rusefillc) [reply firmware/controllers/can/can_dash_ms.cpp:21]: it's a balance between design risks and costs. the code here is not on the mission critical path so we can totally afford x5 CPU consumption for 20% readability improvement.

humans are expensive, CPU is cheap

Comment (mck1117) [reply firmware/controllers/can/can_dash_ms.cpp:21]: Using `outputChannels` is not cheaper at all. If TS isn't connected, there shouldn't be a requirement to populate it at all! There is no guarantee that it is populated, in fact.

**Pull request merged**